camera:

  config_camera: [4306.37205655706, 0.0, 802.1519317540232, 
                     0.0, 4307.411849731741, 479.2531644403244, 
                     0.0, 0.0, 1.0]
  camera_width: 1920
  camera_height: 1200

config_tf: [-0.1, 1.0, 0.0, 1.46,0.052, -0.05]

config_tf_long: [0.2, 0.4, 0.0, 1.62, 0.1, -0.1]

iou_key: 0.2

car_width: 500
car_height: 500

anchor:
  scales: [32, 64, 128]
  ratios: [0.5, 1.0, 2.0]
